schema/data/project-page-data.yaml

```diff
@@ -0,0 +1,48 @@
+
+id: "http://dev.app.regen.com/projects/38"
+marketType: rfs:ComplianceMarket
+bioregion:
+  - "Amazon Basin"
+biomeType:
+  - "TropicalForest"
+watershed:
+  - "Amazon River"
+subWatershed:
+  - "Upper Amazon"
+ecosystemType:
+  - rfs:CloudForest
+offchainCreditsInfo:
+  creditsRegistered:
+    numericValue: 1000
+    unit: unit:HA
+  creditsAvailable:
+    numericValue: 800
+    unit: unit:HA
+  creditsRetired:
+    numericValue: 200
+    unit: unit:HA
+projectDuration: "P1Y"
+projectDurationMinimum: "P6M"
+projectDurationMaximum: "P2Y"
+ecologicalConnectivityIndex:
+  numericValue: 75.5
+  unit: unit:UNITLESS
+socialCulturalIndex:
+  numericValue: 85.0
+  unit: unit:UNITLESS
+environmentalAuthority:
```

Comment:
I'm wondering whether we should include the new project partners in here as well as shown on: https://www.figma.com/design/brkxGV5qNOkZUp0YQl1cxO/Terrasos-Phase-1?node-id=500-31174&node-type=frame&m=dev
Historically, we've been storing this data both as off-chain relationship on `project` table (`developer_id`, `verifier_id`) and in the on-chain project metadata.
I guess we need to decide here if we want to define new db relationships (and tables in case it's 1 to many) for those new roles, or just store them in jsonld for simplicity

Reply:
I've added `projectOperator`, `projectMonitor`, and `projectOwner`, @clevinson and I chatted about the existing `projectVerifier` and `projectDeveloper` fields and decided to omit them, so we can handle them with the existing `project` table solution, does this sound good to you @blushi ?

Reply:
Well historically, we put those stakeholders both in the on chain project metadata because this is important project info and as offchain project relationships to be able to map it to existing accounts if any. I guess we should keep things consistent here.
In any case, in the code, we always look at both sources of data to display those existing stakeholders.
So I would suggest that we keep the `projectVerifier` and `projectDeveloper` fields in the on chain metadata, to keep things consistent, and we can also provide the on chain project dedicated fields if there's any account profile we should link to.
I'm fine not building new db columns/tables to store the new `projectOperator`, `projectMonitor` and `projectOwner` for now to save time and if we think they won't map to actual account, so ok to have those new fields only in the project metadata.

Reply:
Also can there be multiple `projectMonitor`s? cc/ @clevinson @erikalogie

Reply:
@blushi I've asked Malu from Terrasos and our science team about this

Reply:
Malu says that it is possible to have more than one monitor, though extremely uncommon

Reply:
I just added back the `projectDeveloper` and `projectVerifier` fields to the metadata. For now I left `projectMonitor` as a single field